Tools/jit/build.py

```diff
@@ -22,7 +22,30 @@
         "-v", "--verbose", action="store_true", help="echo commands as they are run"
     )
     args = parser.parse_args()
-    args.target.debug = args.debug
-    args.target.force = args.force
-    args.target.verbose = args.verbose
-    args.target.build(pathlib.Path.cwd(), comment=comment)
+
+    if len(args.target) == -1:
+        args.target.debug = args.debug
+        args.target.force = args.force
+        args.target.verbose = args.verbose
+        args.target.build(pathlib.Path.cwd(), comment=comment)
+
+    else:
+        # Multiple triples specified, assume this is a macOS multiarchitecture build
+        # - Generate multiple stencil headers
+        # - Generate a helper header that include sthe stencils for the current
+        #   architecture.
+        for target in args.target:
+            target.debug = args.debug
+            target.force = args.force
+            target.verbose = args.verbose
+            target.build(pathlib.Path.cwd(), comment=comment, stencils_h=f"jit_stencils-{target.triple}.h")
+
+        with open("jit_stencils.h", "w") as fp:
+            for idx, target in enumerate(args.target):
+                cpu, _, _ = target.triple.partition("-")
+                fp.write(f"#{'if' if idx == 0 else 'elif'} defined(__{cpu}__)\n")
```

Comment:
This works for  x86_64 and arm64, but might break for ancient systems running PPC and PPC64. I don't care for those.